core/primitives/tipset/load.hpp

```diff
@@ -81,7 +80,15 @@ namespace fc::primitives::tipset {
     std::map<TipsetKey, uint64_t> map_cache;
     std::vector<LoadNode> tipset_cache;
     uint64_t capacity;
-    uint64_t begin_index;
-    uint64_t end_index;
+    uint64_t begin_index{};
+    uint64_t end_index{};
+  };
+
+  struct PutBlockHeader {
+    virtual ~PutBlockHeader() = default;
+    virtual void put(const CbCid &key, BytesIn value) = 0;
   };
+  CID put(const IpldPtr &ipld,
```

Comment:
Looks like testutil

Reply:
it is not, it is used inside real components, so can't be in testutils